fastlane/lib/fastlane/actions/push_git_tags.rb

```diff
@@ -23,6 +27,15 @@ def self.description
 
       def self.available_options
         [
+          FastlaneCore::ConfigItem.new(key: :force,
+                                       env_name: "FL_PUSH_GIT_FORCE",
+                                       description: "Force push to remote. Defaults to false",
+                                       is_string: false,
+                                       default_value: false),
+          FastlaneCore::ConfigItem.new(key: :remote,
+                                       env_name: "FL_GIT_PUSH_REMOTE",
+                                       description: "The remote to push to. Defaults to `origin`",
+                                       default_value: 'origin')
```

Comment:
That makes sense, I was just following https://github.com/fastlane/fastlane/blob/master/fastlane/lib/fastlane/actions/push_to_git_remote.rb#L58

I'll make the change.